packages/qulacs/tests/qulacs/circuit/test_convert_circuit.py

```diff
@@ -155,21 +165,25 @@ def test_convert_u_gate() -> None:
 def test_convert_pauli_gate() -> None:
     g = gates.Pauli((11, 7, 13), (2, 3, 1))
     converted = convert_gate(g)
-    expected = qulacs.gate.Pauli((11, 7, 13), (2, 3, 1))
+    expected = qulacs.gate.Pauli(cast_to_list((11, 7, 13)), cast_to_list((2, 3, 1)))
```

Comment:
In this case we can simply use lists.
(My past comment about favoring casting is to reduce performance overhead of object conversion when a tuple is already given)